### PR TITLE
make ruko happy / add a cautery to CentCom

### PR DESCRIPTION
**Pull request by bloons3**
## About The Pull Request

Adds a cautery to CentCom

## Why It's Good For The Game
image

## Changelog
:cl:
add: Adds a cautery and surgical drill to CentCom
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
Bloons fixing the real problems with the game and dealing with my pointless salt.

**Review by Archanial — DISMISSED** [review]

**Comment by MCterra10** [comment]
issue is gamebreaking, can we speedmerge this fix? maintainers?

File: _maps/map_files/generic/CentCom.dmm [hunk 1]
```diff
@@ -10981,6 +10981,7 @@
 /obj/effect/turf_decal/tile/blue{
 	dir = 8
 	},
+/obj/item/cautery,
```
Comment (park66665) [inline _maps/map_files/generic/CentCom.dmm:10984]:
```suggestion
/obj/item/cautery,
/obj/item/surgicaldrill,
```

Heads up, CentCom surgery also lacks surgical drill.

**Review by park66665 — APPROVED** [review]
All clear. This PR is good to go.

**Pull request merged**